codespell_lib/data/dictionary.txt

```diff
@@ -4346,39 +4351,56 @@ geoemtries->geometries
 geoemtry->geometry
 geographich->geographic
 geographicial->geographical
+geomery->geometry
+geometites->geometries
+geometrc->geometry
 geometrician->geometer
 geometricians->geometers
 geometrie->geometry
 geometrys->geometries
 geomitrically->geometrically
 geomtry->geometry
 geomtrys->geometries
+geraphics->graphics
 gerat->great
+gererate->generate
+gererated->generated
+gernerate->generate
 gess->guess
 get's->gets
 get;s->gets
 geting->getting
 gettetx->gettext
 Ghandi->Gandhi
+ghraphic->graphic
+gioen->given
+gird->grid
```

Comment:
The word _gird_ is valid, this correction should be removed.

Reply:
Removed this entry in favor of https://github.com/lucasdemarchi/codespell/pull/479/files#diff-3bddc99f1f6dffc98ea11be5a5a31ae9R4945